tokio/tests/rt_unstable_metrics.rs

```diff
@@ -150,6 +151,28 @@ fn remote_schedule_count() {
     assert_eq!(1, rt.metrics().remote_schedule_count());
 }
 
+#[test]
+fn worker_thread_id() {
```

Comment:
(No need to put everything you test related to thread ids in a single test. You can make more than one.)